static/sass/components/_post-content.scss

```diff
@@ -1,26 +1,43 @@
 // Post Content
 // ============
 
-
 .section-title {
   margin-top: 0;
 }
 
 .title {
-  @include font-family('alt');
-  font-size: calc(1.5rem + 1.75vw);
-  line-height: 1.25;
+  @include title;
   margin: size('gutter') auto;
   max-width: size('full-content');
   padding: 0 size('gutter');
 
+  [data-location='work'] & {
```

Comment:
No, I had this before you and I talked about not having multiple title sizes. Forgot to go in and remove.